### PR TITLE
added to the lineup

### PR DESCRIPTION
**Pull request by whichlight**
If you're submitting a PR for a talk or a performance, here's the format you should follow:

```yml
- 7:45 PM:
    type: Talk
    emoji: 📞
    title: 'Your Title Here'
    description: 'Your description here.'
    person:
      name: Foo Bar
      twitter: foobar
```

I think i did it right I'm not sure, let me know if more info needed. thanks!

### CODE REVIEW

File: models/calendar.yml [hunk 1]
```diff
@@ -294,7 +304,7 @@
           name: Ge Wang
           twitter: gewang
 
-    - 9:00 PM:
+    - 9:00 PM:w
```
Comment (TheAlphaNerd) [inline models/calendar.yml]: I think this may have been an accident

**Comment by TheAlphaNerd** [comment]
One small nit mentioned above otherwise LGTM

**Comment by billyroh** [comment]
This looks good! But could you move it to the spot in April? Right now it's in June :sweat_smile:

**Comment by whichlight** [comment]
oh dear, my bad! i will try this agian.

On Mon, Mar 28, 2016 at 4:45 PM, Billy Roh <notifications@github.com> wrote:

> This looks good! But could you move it to the spot in April? Right now
> it's in June [image: :sweat_smile:]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wafflejs/wafflejs.github.io/pull/157#issuecomment-202574475>
>



-- 
Kawandeep Virdee • www.whichlight.com • @whichlight

**Pull request closed without merging**